profile.md

```diff
@@ -0,0 +1,173 @@
+for convinience. If there are any inconsistencies between their definitions in
+this document and the [Open Service Broker API specification](spec.md), then
+the [Open Service Broker API specification](spec.md) SHALL take precidence.
+
+- *Platform*: The software that will manage the cloud environment into which
+  Applications and Service Brokers are provisioned.  Users will not directly
+  provision Services from Service Brokers, rather they will ask the Platform
+  (ie. their cloud provider) to manage Services and interact with the
+  Service Brokers for them.
+
+- *Service*: A managed software offering that can be used by an Application.
+  Typically, Services will expose some API that can be invoked to perform
+  some action. However, there can also be non-interactive Services that can
+  perform the desired actions without direct prompting from the Application.
+
+- *Service Broker*: Service Brokers manage the lifecycle of Services. Platforms
+  interact with Service Brokers to provision, bind and delete Service
+  Instances.
+
+- *Application*: The software that will make use of, be bound to, a Service.
+
+## Context Object
+
+In the [Open Service Broker API specification](spec.md) there are certain
+message flows that include a `context` property. This property is defined
+as an opaque JSON object that is meant to contain contextual information
+about the environment in which the Platform or Application is executing.
+For example, it might include the organizational information (eg. a
+Cloud Foundry `organization` GUID) in which the Application is owned.
+
+While the `context` property is defined as an opaque JSON object, in practice,
+it is often useful and necessary for there to be an agreed upon set of
+properties to ensure a common understanding of this data between the
+Platform and the Service Brokers.
+
+While use of this profile is OPTIONAL, an implementation is not compliant
+with this profile if it fails to satisfy one or more of the MUST, SHALL
+or REQUIRED level requirements defined herein.
+
+### Context Object Properties
+
+The list of properties within the Context Object can vary depending on
+which Service Broker API is being invoked and which Platform is being used.
+This section will define those properties and specify when each is meant
+to be used.
+
+Aside from the Platform specific properties, defined in the following
+sections, there is one common property called `platform` that
+MUST also appear within `context` to indicate which Platform is being used.
+
+The `platform` property MUST be a `string` and serialized as follows:
+```
+"platform" : "platform-string-here"
+```
+
+Each section below will define the `platform` value that MUST be used based
+on the Platform and the set of additional properties that MUST be present.
+
+#### Cloud Foundry
+
+*`platform` Property Value*: `cloudfoundry`
+
+The following properties are defined for usage within a Cloud Foundry
+deployment:
+
+- `organization_guid`<br>
```

Comment:
I think so - I couldn't figure out how to get md to add a break there w/o also adding a blank line. But if it bothers people I can live with the blank line.